bin/speech_to_text_watcher

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+# Continually monitor the speech-to-text worker's "done" queue and report job status to SDR
+
+require 'socket'
+
+require_relative '../config/boot'
+
+# Set log level with LOG_LEVEL; share the logger across the stack
+progname = 'speech_to_text_watcher'
+logger = Logger.new(STDOUT, progname:, level: ENV['LOG_LEVEL'] || 'INFO')
+host = Socket.gethostname
+stt_create_done_handler = Dor::TextExtraction::SpeechToTextCreateDoneHandler.new(host:, progname:, logger:)
+
+
+sqs_watcher = Dor::TextExtraction::SqsWatcher.new(role_arn: Settings.aws.speech_to_text.role_arn,
+                                                   queue_url: Settings.aws.speech_to_text.sqs_done_queue_url)
+sqs_watcher.poll do |done_msg|
+  logger.debug("sqs_watcher.poll block received done_msg: #{done_msg}")
+  stt_create_done_handler.process_done_message(done_msg)
+end
```

Comment:
how does the watcher get started and stay running?  do we need some hooks in the cap deploy like we do for the abbyy_watcher?

Reply:
oh, possibly, i had overlooked the cap deploy hooks.  there is a corresponding [puppet PR for keeping things running via systemd](https://github.com/sul-dlss/puppet/pull/11366), which i might pare back to just stage for now, leaving QA and prod as a separate draft puppet PR to be held till stage gets properly integration tested with this branch deployed.

Reply:
ok, i see the cap config stuff, i think:
* https://github.com/sul-dlss/common-accessioning/blob/74a2910afe57c401f920add0c21543ddbc2405c6/config/deploy.rb#L46-L47
* https://github.com/sul-dlss/common-accessioning/blob/74a2910afe57c401f920add0c21543ddbc2405c6/config/deploy/stage.rb#L13 (and similar for QA and prod)
* https://github.com/sul-dlss/common-accessioning/blob/main/lib/capistrano/tasks/abbyy_watcher_systemd.cap#L4-L11

i had seen and forgotten the first two things, but had totally missed the third of those.

thinking about it... 
* i think adding the capistrano hooks before the puppet PR is merged would likely break deployment (or at least lead to warnings we don't actually need to worry about), because the cap hooks would refer to some systemd stuff that isn't actually set up yet.
* and i think merging the puppet PR that assumes this watcher script is in place before it's merged to `main` would cause unnecessary warnings, at least, since there'd be attempts to restart a non-existent service.

so my inclination is to ticket the cap config stuff, get this PR merged, then get the puppet PR merged for all envs, then do the ticket for the cap config stuff.

or, maybe we could do all the cap stuff except for the `before 'deploy:publishing', 'abbyy_watcher_systemd:restart'` in `deploy.rb`?

still might be nice to get this merged first.  i'd propose:
* i finish addressing other feedback on this PR
* i deploy this branch to stage and start the daemon manually in a screen session and run the integration test to see if things at least get to marking `stt-create` complete
* i ticket the cap hook stuff for this codebase
* we get this PR merged
* we get the puppet changes merged for all envs
* we unblock the ticket for adding cap hooks to this codebase

Reply:
https://github.com/sul-dlss/common-accessioning/issues/1413